src/dots.jsx

```diff
@@ -25,8 +25,9 @@ export var Dots = React.createClass({
       slidesToScroll: this.props.slidesToScroll
     });
 
-    var dots = Array.apply(null, {length: dotCount}).map((x, i) => {
+    var dots = [];
 
+    for (var i=0; i<dotCount; i++) {
```

Comment:
Assume one has included es5-shim for IE8 supporting, Array.map should be safe. The problem is within `.apply`. It should be: `var dots = Array.apply(null, Array(dotCount)).map(...`
Tested with native IE8, it works.